GameData/FerramAerospaceResearch/FAR.version

```diff
@@ -10,7 +10,7 @@
     "MAJOR" : 0,
     "MINOR" : 15,
     "PATCH" : 9,
-    "BUILD" : 5
+    "BUILD" : 6
```

Comment:
No need to change the version, I will do it when I publish a new release

Reply:
Roger, will set it back to 5.